code/client/launcher/Main.cpp

```diff
@@ -295,16 +297,26 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine
 
 		wcscat_s(systemPath, dll);
 
-		LoadLibrary(systemPath);
+		if(!LoadLibrary(systemPath))
+		{
+			printf("Could not initialize ");
+			wprintf(dll);
+			printf(".\n");
+		}
 	}
 
-	LoadLibrary(MakeRelativeCitPath(L"dinput8.dll").c_str());
-	LoadLibrary(MakeRelativeCitPath(L"steam_api64.dll").c_str());
+	if(!LoadLibrary(MakeRelativeCitPath(L"dinput8.dll").c_str()))

```

Comment:
With these extra lines of code it might make more sense to just iterate over an initializer list?